nbs/examples/ElectricityPeakForecasting.ipynb

```diff
@@ -0,0 +1,14449 @@
+{
```

Comment:
<p>A even more general intro would be good. Something like:</p><p>"Prediciting peaks in different markets is useful. In the electricity market, consuming electricity at peak demand is penalized with higher tarifs. When an individual or company consumes electricity when its most demanded, regulators calls that CP.</p><p> </p><p>In Texas (and other markets) a coincident peak (CP) is the 15-minute.. "</p><p> </p><p>Also, it might be good to justify the use of MSTL:</p><p>"Multiple seasonality is traditionally present in low sampled electricity data. Demand exhibits daily and weekly seasonality. That means that there are clear patterns of electricity consumption for specific hours of the day like 6:00pm vs 3:00am or for specific days like Sunday vs Friday."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done